introduction.md

```diff
@@ -0,0 +1,7 @@
+My name is Carlos Polachini Zanoveli Junior and I was born in Campinas, São Paulo, and raised in Londrina, in the countryside of Paraná. I've completed the highschool at Colégio Marista de Londrina and done a year of preparatory course at Anglo Castelo in Campinas before entered in State University of Campinas (Unicamp), in 2009.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.